web/packages/teleport/src/Discover/Database/SetupAccess/AwsRdsAuthRequirements.tsx

```diff
@@ -0,0 +1,389 @@
+/**
+ * Teleport
+ * Copyright (C) 2025  Gravitational, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import React, { PropsWithChildren, useState } from 'react';
+import styled from 'styled-components';
+
+import { Alert, Box, Flex, Label, Link, Mark, Text } from 'design';
+import { CollapsibleInfoSection } from 'design/CollapsibleInfoSection';
+import { ChevronDown } from 'design/Icon';
+import { SpaceProps } from 'design/system';
+import { H3 } from 'design/Text';
+import { TextSelectCopyMulti } from 'shared/components/TextSelectCopy';
+
+import { Tabs } from 'teleport/components/Tabs';
+import {
+  AwsRdsGuideIds,
+  getRdsIamAuthnHref,
+} from 'teleport/Discover/Overview/Overview';
+import { DatabaseServiceDeploy } from 'teleport/Discover/useDiscover';
+import { DiscoverGuideId } from 'teleport/services/userPreferences/discoverPreference';
+
+export function AwsRdsAuthRequirements({
```

Comment:
Seems like something is adding additional spacing here, do you know where this is coming from?
<img width="642" alt="image" src="https://github.com/user-attachments/assets/e88b7c3a-7e2e-4f3f-aaf0-8066e14b8955" />